### DISCUSSION TITLE
I am getting the json structure in front end when using the columns blocks

### DISCUSSION THREAD

**Discussion by amielarieti** [WebDevStudios/nextjs-wordpress-starter#374] [General]
I am getting the json structure  query in front end when using the columns blocks
I am not getting actual errors but for some reason the blocks are not rendering as output. Screenshot here
https://prnt.sc/11l557o

**Comment by sloyer** [comment]
I have ran into this problem, typically its with a block not supported by this Starter.
Thought it looks like yours is just a header and i tried locally on mine and had no issues. 

Have you deleted the Block and tried again? Maybe something funky happened in the data.

**Reply by gregrickaby** [reply]
@amielarieti 

@sloyer Is correct. You're seeing raw JSON on purpose, that's to let you know that you've got an unsupported block in your page or post. See https://github.com/WebDevStudios/nextjs-wordpress-starter/blob/main/functions/wordpress/blocks/displayBlock.js#L128

What's great about seeing JSON, is now you know the shape of the block and create your own support for it in the starter. 

See https://webdevstudios.com/2021/04/20/quick-tip-gutenberg-blocks-as-json/ for more info.
